test/manifest

```diff
@@ -8,5 +8,5 @@ splash_screen_sd=
 splash_color=#000000
 splash_min_time=0
 major_version=1
-minor_version=0
```

Comment:
We don't want to change the manifest version, we want to change the version in the /VERSION file.

Reply:
Updated Bryan. Thanks for explaining :)